packages/jest-serializer-make-styles/src/index.ts

```diff
@@ -0,0 +1,22 @@
+import { DEFINITION_LOOKUP_TABLE, RULE_CLASSNAME_INDEX, RTL_PREFIX } from '@fluentui/make-styles';
+
+export function print(val: string) {
+  const undesiredParts: string[] = [];
+  const lookupRegex = new RegExp(`${Object.keys(DEFINITION_LOOKUP_TABLE).join('|')}`, 'g');
```

Comment:
Can you please add comments to regexps? To provide some understanding for what they are looking.